susie/built.sbt

```diff
@@ -0,0 +1,70 @@
+val Versions = new {
+  val Aggregator = "0.3.4-SNAPSHOT"
```

Comment:
Change this to 3.5 which I just released

Reply:
done!